tests/resources/rebuild_example_data

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+from code import InteractiveConsole
+import readline  # noqa
```

Comment:
this enables readline support for prompts in python like `input` and the interactive console.